src/components/v5/shared/ProgressBar/ProgressBar.tsx

```diff
@@ -5,19 +5,20 @@ import { type ProgressBarProps } from './types.ts';
 
 const ProgressBar: FC<ProgressBarProps> = ({
   progress,
+  progressLabel,
```

Comment:
since we've added a `progressLabel` here, I think we can skip `labelClassName`, just make `progressLabel` a `JSX.Element` type, and render it or the `progress` variable. Then in all the places you want to use `labelClassName` just do `<span className="text-xs">{progress}%</span>`